sponsor-dapp/package.json

```diff
@@ -17,6 +17,10 @@
     "web3": "1.0.0-beta.37"
   },
   "scripts": {
+    "start": "react-scripts start",
```

Comment:
Maybe I'm just reading the diff incorrectly, but are "start" and "build" duplicated?

Also, wdyt about having an explicit MODE for the normal case as well, rather than the value being "monitoring" vs undefined?

Reply:
You're correct, they are duplicated. I fixed that.

I thought about this, and I think it makes sense in principle. The downsides are:
- We would have to have two new commands for building the `default` version of the dapp. 
- We would have to always provide an env variable if we were to use the usual `npm start` and `npm run build` commands.

My suggestion is to add in new commands that pass `default` as the mode, but running without the env variable has the same behavior. WDYT?

Reply:
@ptare bump.

Reply:
Sorry I missed this PR, LGTM.